Exiled.API/Features/Roles/FpcRole.cs

```diff
@@ -87,6 +98,24 @@ public float CrouchingSpeed
             set => FirstPersonController.FpcModule.CrouchSpeed = value;
         }
 
+        /// <summary>
+        /// Gets or sets the <see cref="Player"/> velocity.
+        /// </summary>
+        public Vector3 Velocity
+        {
+            get => FirstPersonController.FpcModule.Motor.Velocity;
+            set => FirstPersonController.FpcModule.Motor.Velocity = value;
```

Comment:
Has changing their velocity been tested? What does it actually affect? Does it cause desync issues?